content/packit-as-a-service.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Packit-as-a-Service"
+date: 2019-06-22
+draft: false
+---
+
+Packit-as-a-Service is a GitHub App (https://developer.github.com/apps/about-apps/).
+
+### Easy to use
+
+Using Packit Service is very straightforward: add one config file to your repository
+([see .packit.yaml format](https://github.com/packit-service/packit/blob/master/docs/configuration.md)),
+followed by an RPM spec file and you're good to go. Besides the service, packit is also as a tool
+([see packit README.md](https://github.com/packit-service/packit/blob/master/README.md)),
+so you can always try things locally on your own.
+
+Packit Service validates your pull requests by building your software in Fedora OS.
+Once the builds are done, packit lets you know how to install the
+RPMs inside your environment so you can try the changes before merging them.
+
+<!--more-->
+
+## Steps how to integrate Packit-as-a-Service into your projects or organizations
+
+There are two ways to install Packit-as-a-Service GitHub app into your repository.
+
+### Install from Packit-as-a-Service page
+1. From the [Packit-service page](https://github.com/apps/packit-as-a-service) page,
+in the left sidebar, click "Install App"
+2. Click "Install" next to the organization or user account containing the correct repository
+3. Install the "Packit-as-a-Service" on all repositories or to certain repositories
+
+### Install from GitHub Marketplace
```

Comment:
since we won't be on marketplace in following days, I'd comment this section out